src/ComboboxInput/__snapshots__/ComboboxInput.test.js.snap

```diff
@@ -14,8 +14,12 @@ exports[`<ComboboxInput /> create combobox input 1`] = `
         className="fd-popover__control"
       >
         <div
+          aria-haspopup={true}
           className="fd-input-group"
           onClick={[Function]}
+          onKeyPress={[Function]}
+          role="button"
```

Comment:
`space` keypress is consumed by this `div` wrapper and hence it's not possible to type out an actual `space` in the `input` below. The dropdown `button` also behaves weirdly when we press `space`.